App/Screens/Account.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { View } from "react-native";
+import { withNavigation } from 'react-navigation';
+import { Card, Button, Text } from "react-native-elements";
+import { onSignOut } from "../Config/auth";
+
+
+export class Account extends Component {
+
+
+  //export default ({ navigation }) => (
+  render(){
+
+    const { navigation } = this.props;
+
+    return(
+      <View style={{ paddingVertical: 20 }}>
+        <Card title="John Doe">
+          <View
+            style={{
```

Comment:
When you have a multiline style object, you usually want to move it to the bottom of the function using `StyleSheet.create()` instead of inlining them directly. Try to keep the style prop a single line if you can.